spec.md

```diff
@@ -673,10 +679,82 @@ Errors](#service-broker-errors).
 
 #### Body
 
-All response bodies MUST be a valid JSON Object (`{}`). This is for future
-compatibility; it will be easier to add fields in the future if JSON is
-expected rather than to support the cases when a JSON body might or might not
-be returned.
+For success responses, the following fields are valid.
+
+| Response field | Type | Description |
+| --- | --- | --- |
+| state* | string | Valid values are `in progress`, `succeeded`, and `failed`. While `"state": "in progress"`, the platform SHOULD continue polling. A response with `"state": "succeeded"` or `"state": "failed"` MUST cause the platform to cease polling. |
+| description | string | A user-facing message displayed to the platform API client. Can be used to tell the user details about the status of the operation. If present, MUST be a non-empty string. |
+
+\* Fields with an asterisk are REQUIRED.
+
+```
+{
+  "state": "in progress",
+  "description": "Creating service (10% complete)."
+}
+```
+
+## Polling Last Operation for Service Bindings
+
+When a broker returns status code `202 Accepted` for [Binding](#binding) or
+[Unbinding](#unbinding), the platform will begin polling the
+`/v2/service_instances/:instance_id/service_bindings/:binding_id/last_operation`
+endpoint to obtain the state of the last requested operation. The broker
+response MUST contain the field `state` and MAY contain the field `description`.
+
+Valid values for `state` are `in progress`, `succeeded`, and `failed`. The
+platform will poll the `last_operation` endpoint as long as the broker returns
+`"state": "in progress"`. Returning `"state": "succeeded"` or
+`"state": "failed"` will cause the platform to cease polling and, in the case of
+a [Binding](#binding) request, information on the service binding can then
+immediately be fetched using the
+[Fetching a Service Binding](#fetching-a-service-binding) endpoint. The value
```

Comment:
But what if the catalog endpoint returns `"binding_retrievable": false`? Do we still support async bindings (without being able to fetch any result)?

Reply:
I think that we should explicitly add that if a binding supports async operations, it must have `"binding_retrievable": true`.

Reply:
The wording with `binding_retrievable` is a bit ambiguous. It says:

```If `"binding_retrievable" :true` is declared for a service in the Catalog endpoint, brokers MUST support this endpoint for all plans of the service.```

It doesn't say that having `"binding_retrievable": false` means the endpoint is unavailable for all plans, so you could potentially have some plans able to support it and some not, and thus some plans able to be asynchronously bound to and some not.

I think we should tighten the language on the `binding_retrievable` definition.

Reply:
I think this is sensible, thanks!